inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
-// Copyright 2015
-// Ubiquitous Knowledge Processing (UKP) Lab and FG Language Technology
-// Technische Universität Darmstadt
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-// http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+Getting Started
 
-= Getting started
+This Getting-Started-Guide shall assist new users to install and start
+and work with INCEpTION. It gives a quick overview (estimated time for
+reading only: approx. 15 minutes) on the main issues in order to get
+familiar with the tool and does not incude special cases and details due
+to simplicity and focus on the first steps. For further reading on any
+issue, see our main documentation.: LINK EINFÜGEN. For quick overviews,
+also see our tutorial videos (Topics: Introduction, Overview,
+Recommender Basics, Entity Linking):
+https://www.youtube.com/playlist?list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-
 
-In order to run {product-name}, you need to have Java installed on your system in version 8 or 
-higher. If you do not have Java installed yet, please the latest link:https://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle Java] or link:https://adoptopenjdk.net[OpenJDK].
+ +
+_Introduction_
```

Comment:
I do not like the many bullet points, I want flow text

Reply:
I dislike much text in descriptions and highly prefer bullet points. It seems to be a matter of taste.
@UWinch , @reckart  What do you think?

Reply:
Trade-off between space and readability.

Pro bullet-point:
- stronger emphasis on the bullet point messages - in particular for enumerations (if you consider these also under "bullet points)
- makes a lot of sense if the items consist of more than one sentence each

Contra bullet-point: 
- if the items are very short
- takes up more space

Bullet points shouldn't be used as a way of organizing longish paragraphs or even sections - section headers are more sensible in these cases.

Reply:
I mean it especially for later, when there are full paragraphs as bullet points. You can check it in the rendered documentation.

Reply:
So is this about standard listings (woth explanations like in "Anatomy of an annotation project"; or really short and standard like in the sub point right there: "Couration") or about structuring whole paragraphs whoch are one step to do each?

I think it makes sense to show clearly where one step starts and ends for good orientation (especially when returning to the document after having finished one step).

Reply:
Generate the HTML and observer that you almost never have paragraphs, but almost always use bullet points. This is true for almost all sections of this guide. This is really unusual imho and does not fit to the rest of the documentation.